docs/technical-documentation.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Technical documentation
+category: Technical documentation
+---
+
+# Architecture
+
+## Micro-services architecture
+
+Here is an overview of PowSyBl micro-services architecture and corresponding repositories:
+
+<object type="image/svg+xml" data="./architecture/images/micro-services-architecture.svg" >
+</object>
+
+### Case server
+
+This services is responsible from raw network data storage. It relies on a file system to store its data and allows uploading and fetching network data for any network data format supported by PowSyBl (CGMES, UCTE, XIIDM, etc).
+
+### Network store server
+
+As most of the feature developed in PowSyBl relies on IIDM API for accessing to network data, the idea of the service is to re-implement the IIDM API so that it is better integrated in a micro-services typical architecture. Instead of an in memory implementation like in PowSyBl core repository, this implementation is backed by a Cassandra database. Network model is stored in the database in a detailed way (one table per equipment), so that we can query only the amount of needed data and ensure a good performance for a simple switch position change (few data), a single line diagram rendering (substation wide data) or a loadflow run (network wide data). Network data should only be accessed from this REST Web service using the provided Java client (the one which implement IIDM API) and never directly from the REST interface.
```

Comment:
Typo:
- *As most* vs Almost?
- access sth --> for accessing network data
- so that it is --> to be better
- in memory --> in-memory?

Reply:
Done